### PR TITLE
MAILBOX-351 Remove GlobalRegistration

### PR DESCRIPTION
**Pull request by chibenwa**
Non controversial commits of https://github.com/linagora/james-project/pull/1948

### CODE REVIEW

File: mailbox/tools/indexer/src/main/java/org/apache/mailbox/tools/indexer/ReIndexerPerformer.java [hunk 1]
```diff
@@ -109,31 +109,24 @@ public ReIndexerPerformer(MailboxManager mailboxManager,
...
             })
             .reduce(Task::combine)
             .orElse(Task.Result.COMPLETED);
     }
 
-    private Task.Result reIndex(MailboxPath path, MailboxSession mailboxSession, ReprocessingContext reprocessingContext) throws MailboxException {
-        LOGGER.info("Intend to reindex {}", path.asString());
-        Mailbox mailbox = mailboxSessionMapperFactory.getMailboxMapper(mailboxSession).findMailboxByPath(path);
+    private Task.Result reIndexSingleMailbox(MailboxId mailboxId, ReprocessingContext reprocessingContext) throws MailboxException {
+        LOGGER.info("Intend to reindex {}", mailboxId);
```
Comment (trantienduchn) [inline mailbox/tools/indexer/src/main/java/org/apache/mailbox/tools/indexer/ReIndexerPerformer.java]: s/mailboxId/mailboxId.serialize() ?

File: mailbox/tools/indexer/src/main/java/org/apache/mailbox/tools/indexer/ReIndexerPerformer.java [hunk 2]
```diff
@@ -145,7 +138,7 @@ public ReIndexerPerformer(MailboxManager mailboxManager,
                 .reduce(Task::combine)
                 .orElse(Task.Result.COMPLETED);
         } finally {
-            LOGGER.info("Finish to reindex {}", path.asString());
+            LOGGER.info("Finish to reindex {}", mailboxId);
```
Comment (trantienduchn) [inline mailbox/tools/indexer/src/main/java/org/apache/mailbox/tools/indexer/ReIndexerPerformer.java]: idem

File: mailbox/tools/indexer/src/main/java/org/apache/mailbox/tools/indexer/ReIndexerPerformer.java (hunk 1 above)
Comment (chibenwa) [reply mailbox/tools/indexer/src/main/java/org/apache/mailbox/tools/indexer/ReIndexerPerformer.java]: Okay but I don't want just to have just a UUID that could be anything, it should be clear it is a mailbox

**Comment by chibenwa** [comment]
test this please

Crash when playing Cassandra RabbitMQ MPT tests

**Comment by chibenwa** [comment]
As argued here https://github.com/linagora/james-project/pull/1948#discussion_r236070485 it is not that easy, very sadly ...

**Comment by hoangdat** [comment]
Build triggered. sha1 is merged.

**Comment by hoangdat** [comment]
Build started sha1 is merged.

**Comment by hoangdat** [comment]
Build finished. No test results found.

**Comment by hoangdat** [comment]
Build triggered. sha1 is merged.

**Comment by hoangdat** [comment]
Build started sha1 is merged.

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**